### PR TITLE
Fix manual create sourcing

### LINKED ISSUE #33: 8.0: cannot create a source manually
Install `logistic_requisition` alone, create a `logistic.requisition.line` and try to add a source.

You cannot fill in the currency, but when you try to save, you get an error saying that the currency is missing.

### PR DESCRIPTION
**Pull request by lepistone**
Fixes #33.

### CODE REVIEW

File: logistic_requisition/model/logistic_requisition.py [hunk 1]
```diff
@@ -528,13 +528,20 @@ def _get_total_cost(self):
...
-            total_cost_company += from_curr.compute(total_cost, to_curr,
-                                                    round=False)
-            line.amount_total_company = total_cost_company
+            requisition = line.requisition_id
+            # if we do not know the requisition, then this method is probably
+            # called like an on_change, before saving. In that case, we return
+            # instead of crashing. The result of that is that the
+            # amount_total_company field will be updated only when we save.
+            # It might be that that can be improved by improving the view. This
+            # check will pose no problem in that case.
```
Comment (gurneyalex) [inline logistic_requisition/model/logistic_requisition.py]: should we log something in that case?

Comment (lepistone) [inline logistic_requisition/model/logistic_requisition.py]: agreed: that should be a **warn**.

File: logistic_requisition/model/logistic_requisition.py [hunk 2]
```diff
@@ -528,13 +528,20 @@ def _get_total_cost(self):
...
-
-            total_cost_company += from_curr.compute(total_cost, to_curr,
-                                                    round=False)
-            line.amount_total_company = total_cost_company
+            requisition = line.requisition_id
+            # if we do not know the requisition, then this method is probably
+            # called like an on_change, before saving. In that case, we return
+            # instead of crashing. The result of that is that the
+            # amount_total_company field will be updated only when we save.
+            # It might be that that can be improved by improving the view. This
```
Comment (rdeheele) [inline logistic_requisition/model/logistic_requisition.py]: s/that that/that (i think)

**Comment by lepistone** [comment]
@rdeheele @gurneyalex done.

**Comment by gurneyalex** [comment]
:+1:

**Pull request merged**